<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GfxEditorTab</class>
 <widget class="QWidget" name="GfxEditorTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>594</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="mainLayout" stretch="0">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QScrollArea" name="editScrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="GfxEditorWidget" name="editorWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>69</width>
         <height>574</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="rightLayoutWidget">
      <layout class="QVBoxLayout" name="rightLayout">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="horizontalSpacing">
          <number>2</number>
         </property>
         <property name="verticalSpacing">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="widthLabel">
           <property name="text">
            <string>Width:</string>
           </property>
           <property name="buddy">
            <cstring>widthCombo</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="widthCombo"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="heightLabel">
           <property name="text">
            <string>Height:</string>
           </property>
           <property name="textFormat">
            <enum>Qt::PlainText</enum>
           </property>
           <property name="buddy">
            <cstring>heightCombo</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="heightCombo"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="colorModeLabel">
           <property name="text">
            <string>Color:</string>
           </property>
           <property name="buddy">
            <cstring>colorModeCombo</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="colorModeCombo"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="formatLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Format:</string>
           </property>
           <property name="buddy">
            <cstring>formatCombo</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="formatCombo"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="entityLabel">
           <property name="text">
            <string>Entity:</string>
           </property>
           <property name="buddy">
            <cstring>entityEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="walkableLabel">
           <property name="text">
            <string>Walkable:</string>
           </property>
           <property name="buddy">
            <cstring>walkableCheck</cstring>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QCheckBox" name="walkableCheck">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="entityEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>5</width>
           <height>5</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>5</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::HLine</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="backgroundLayout">
         <item>
          <widget class="QCheckBox" name="backgroundImageVisibleCheck">
           <property name="text">
            <string>Background image</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="backgroundImageOpacitySlider">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="browseBackgroundImageButton">
           <property name="text">
            <string>Browse...</string>
           </property>
           <property name="icon">
            <iconset resource="../../res/resources.qrc">
             <normaloff>:/resources/fatcow16x16/folder.png</normaloff>:/resources/fatcow16x16/folder.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="frame2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>5</width>
           <height>5</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>5</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::HLine</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exportImageButton">
         <property name="text">
          <string>Export as PNG...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="paletteFrame">
         <property name="frameShape">
          <enum>QFrame::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QVBoxLayout" name="paletteFrameLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="paletteLayout">
            <item>
             <spacer name="paletteLeftSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QGridLayout" name="paletteColorsLayout">
              <item row="5" column="3">
               <widget class="QToolButton" name="brightMagentaButton">
                <property name="styleSheet">
                 <string notr="true">padding: 3px</string>
                </property>
                <property name="icon">
                 <iconset resource="../../res/resources.qrc">
                  <normaloff>:/resources/colors/11_bright_magenta.png</normaloff>:/resources/colors/11_bright_magenta.png</iconset>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonIconOnly</enum>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QToolButton" name="brightBlueButton">
                <property name="styleSheet">
                 <string notr="true">padding: 3px</string>
                </property>
                <property name="icon">
                 <iconset resource="../../res/resources.qrc">
                  <normaloff>:/resources/colors/09_bright_blue.png</normaloff>:/resources/colors/09_bright_blue.png</iconset>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonIconOnly</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QToolButton" name="blackButton">
                <property name="styleSheet">
                 <string notr="true">padding: 3px</string>
                </property>
                <property name="icon">
                 <iconset resource="../../res/resources.qrc">
                  <normaloff>:/resources/colors/00_black.png</normaloff>:/resources/colors/00_black.png</iconset>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonIconOnly</enum>
                </property>
               </widget>
              </item>
              <item row="4" column="3">
               <widget class="QToolButton" name="whiteButton">
                <property name="styleSheet">
                 <string notr="true">padding: 3px</string>
                </property>
                <property name="icon">
                 <iconset resource="../../res/resources.qrc">
                  <normaloff>:/resources/colors/07_white.png</normaloff>:/resources/colors/07_white.png</iconset>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonIconOnly</enum>
                </property>
               </widget>
              </item>
              <item row="5" column="2">
               <widget class="QToolButton" name="brightRedButton">
                <property name="styleSheet">
                 <string notr="true">padding: 3px</string>
                </property>
                <property name="icon">
                 <iconset resource="../../res/resources.qrc">
                  <normaloff>:/resources/colors/10_bright_red.png</normaloff>:/resources/colors/10_bright_red.png</iconset>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonIconOnly</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QToolButton" name="blueButton">
                <property name="styleSheet">
                 <string notr="true">padding: 3px</string>
                </property>
                <property name="icon">
                 <iconset resource="../../res/resources.qrc">
                  <normaloff>:/resources/colors/01_blue.png</normaloff>:/resources/colors/01_blue.png</iconset>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonIconOnly</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QToolButton" name="redButton">
                <property name="styleSheet">
                 <string notr="true">padding: 3px</string>
                </property>
                <property name="icon">
                 <iconset resource="../../res/resources.qrc">
                  <normaloff>:/resources/colors/02_red.png</normaloff>:/resources/colors/02_red.png</iconset>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonIconOnly</enum>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QToolButton" name="greenButton">
                <property name="styleSheet">
                 <string notr="true">padding: 3px</string>
                </property>
                <property name="icon">
                 <iconset resource="../../res/resources.qrc">
                  <normaloff>:/resources/colors/04_green.png</normaloff>:/resources/colors/04_green.png</iconset>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonIconOnly</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QToolButton" name="magentaButton">
                <property name="styleSheet">
                 <string notr="true">padding: 3px</string>
                </property>
                <property name="icon">
                 <iconset resource="../../res/resources.qrc">
                  <normaloff>:/resources/colors/03_magenta.png</normaloff>:/resources/colors/03_magenta.png</iconset>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonIconOnly</enum>
                </property>
               </widget>
              </item>
              <item row="4" column="2">
               <widget class="QToolButton" name="yellowButton">
                <property name="styleSheet">
                 <string notr="true">padding: 3px</string>
                </property>
                <property name="icon">
                 <iconset resource="../../res/resources.qrc">
                  <normaloff>:/resources/colors/06_yellow.png</normaloff>:/resources/colors/06_yellow.png</iconset>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonIconOnly</enum>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QToolButton" name="brightGreenButton">
                <property name="styleSheet">
                 <string notr="true">padding: 3px</string>
                </property>
                <property name="icon">
                 <iconset resource="../../res/resources.qrc">
                  <normaloff>:/resources/colors/12_bright_green.png</normaloff>:/resources/colors/12_bright_green.png</iconset>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonIconOnly</enum>
                </property>
               </widget>
              </item>
              <item row="6" column="2">
               <widget class="QToolButton" name="brightYellowButton">
                <property name="styleSheet">
                 <string notr="true">padding: 3px</string>
                </property>
                <property name="icon">
                 <iconset resource="../../res/resources.qrc">
                  <normaloff>:/resources/colors/14_bright_yellow.png</normaloff>:/resources/colors/14_bright_yellow.png</iconset>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonIconOnly</enum>
                </property>
               </widget>
              </item>
              <item row="6" column="3">
               <widget class="QToolButton" name="brightWhiteButton">
                <property name="styleSheet">
                 <string notr="true">padding: 3px</string>
                </property>
                <property name="icon">
                 <iconset resource="../../res/resources.qrc">
                  <normaloff>:/resources/colors/15_bright_white.png</normaloff>:/resources/colors/15_bright_white.png</iconset>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonIconOnly</enum>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QToolButton" name="cyanButton">
                <property name="styleSheet">
                 <string notr="true">padding: 3px</string>
                </property>
                <property name="icon">
                 <iconset resource="../../res/resources.qrc">
                  <normaloff>:/resources/colors/05_cyan.png</normaloff>:/resources/colors/05_cyan.png</iconset>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonIconOnly</enum>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QToolButton" name="brightCyanButton">
                <property name="styleSheet">
                 <string notr="true">padding: 3px</string>
                </property>
                <property name="icon">
                 <iconset resource="../../res/resources.qrc">
                  <normaloff>:/resources/colors/13_bright_cyan.png</normaloff>:/resources/colors/13_bright_cyan.png</iconset>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonIconOnly</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="paletteRightSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="blinkingLayout">
            <item>
             <spacer name="blinkingLeftSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="blinkingCheck">
              <property name="text">
               <string>Flash</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="blinkingRightSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="previewScrollArea">
         <property name="styleSheet">
          <string notr="true">background-color: black</string>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <widget class="QLabel" name="previewWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>598</width>
            <height>139</height>
           </rect>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GfxEditorWidget</class>
   <extends>QWidget</extends>
   <header>editor/gfx/GfxEditorWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>editScrollArea</tabstop>
  <tabstop>widthCombo</tabstop>
  <tabstop>heightCombo</tabstop>
  <tabstop>colorModeCombo</tabstop>
  <tabstop>formatCombo</tabstop>
  <tabstop>entityEdit</tabstop>
  <tabstop>walkableCheck</tabstop>
  <tabstop>blackButton</tabstop>
  <tabstop>blueButton</tabstop>
  <tabstop>redButton</tabstop>
  <tabstop>magentaButton</tabstop>
  <tabstop>greenButton</tabstop>
  <tabstop>cyanButton</tabstop>
  <tabstop>yellowButton</tabstop>
  <tabstop>whiteButton</tabstop>
  <tabstop>brightBlueButton</tabstop>
  <tabstop>brightRedButton</tabstop>
  <tabstop>brightMagentaButton</tabstop>
  <tabstop>brightGreenButton</tabstop>
  <tabstop>brightCyanButton</tabstop>
  <tabstop>brightYellowButton</tabstop>
  <tabstop>brightWhiteButton</tabstop>
  <tabstop>blinkingCheck</tabstop>
  <tabstop>previewScrollArea</tabstop>
 </tabstops>
 <resources>
  <include location="../../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
